client/src/components/Upload/UploadModalContent.vue

```diff
@@ -167,12 +168,50 @@ export default {
         currentFtp: function () {
             return this.currentUserId && this.ftpUploadSite;
         },
-
+        toData: function (items, history_id) {
+            var data = {
```

Comment:
Shouldn't this be const ? I think generally speaking we'll want to use `const` if there is no reassignment or update and `let` if there is and avoid var as far as possible. In this case you're updating properties of data, which is fine with `const` as well